lib/open_food_network/reports/line_items.rb

```diff
@@ -5,6 +5,7 @@ class LineItems
       def initialize(order_permissions, params, orders_relation = nil)
         @order_permissions = order_permissions
         @params = params
+        complete_not_canceled_visible_orders = CompleteVisibleOrders.new(order_permissions).query.not_state(:canceled)
         @orders_relation = orders_relation || complete_not_canceled_visible_orders
```

Comment:
If we cut these two lines down to just one, like this:
```ruby
@orders_relation = orders_relation || CompleteVisibleOrders.new(order_permissions).query.not_state(:canceled)
```
Then the `CompleteVisibleOrders` won't be instantiated if `orders_relation` is present, right?

Reply:
I did this way to split the long line Rubocop didn't like.